src/hooks/queries/useGetEventList.ts

```
@@ -1,21 +1,18 @@
-// hooks/useEventList.ts
+// hooks/queries/useGetEventList.ts
 import { eventApi } from "@/apis/eventApi";
 import { QueryKey } from "@/constants/queryKey";
 import { EventResponse } from "@/types/dtos/event";
 import { keepPreviousData, useQuery } from "@tanstack/react-query";
```

ESLint

[error] `@tanstack/react-query` import should occur before import of `@/apis/eventApi`

(import/order)